Source/WebKit/Shared/API/APIData.h

```diff
@@ -86,9 +86,9 @@ class Data : public ObjectImpl<API::Object::Type::Data> {
         m_freeDataFunction(const_cast<uint8_t*>(m_span.data()), m_context);
     }
 
-    const unsigned char* bytes() const { return m_span.data(); }
+    const unsigned char* data() const { return m_span.data(); }
```

Comment:
Do we even need this function?

Reply:
I seem to remember there are still a few call sites. I can look into removing it though.